src/main/kotlin/biokotlin/genome/MAFProcessingUtils.kt

```diff
@@ -447,4 +452,321 @@ fun getCoverageIdentityPercentForMAF(mafFile:String, region:String = "all"): Dat
     }.toDataFrame()
     return df
 }
+/**
+ * function to bgzip and create tabix index on a gvcf file
+ *
+ * This method uses the -f option on  bgzip to overwrite any existing files
+ * This handles the case where there already exists a bgzipped file, however
+ * it is still required to have the non-bzipped file present.
+ *
+ * @return the filename of the bgzipped gvcf
+ */
+fun compressAndIndexFile(fileName: String): String {
+
+    try {
+        // First bgzip the file
+
+        // bgzip the file - needed to create index
+        // use the -f option to overwrite any existing file
+        println("bgzipping  file ${fileName}")
+        val gvcfGzippedFile = fileName + ".gz"
+        var builder = ProcessBuilder("conda","run","-n","phgv2-conda",
```

Comment:
We should not be running the phgv2 conda package here.  This will be cross contaminating the two repos with each other.  As now PHGv2 is a dependency on Biokotlin which is a dependency on PHGv2.  I would parameterize this function to take in either a path to bgzip/bcftools through a String array.

Reply:
right!  This is cut/paste issue - I will change